### ISSUE TITLE
[Q] questions about summaries

### ISSUE THREAD

**Issue by ywpkwon** [tensorpack/tensorpack#993]
Hi, I'm enjoying tensorpack, and trying a semantic segmentation example. (I know that MaskRCNN and HED are there, which are very helpful, but more original semantic segmentation.)

I like using tensorboard, so I'd like to ask a few questions.

1. Tensorpack generally seems to record summaries and also run callbacks such as `GPUUtilizationTracker`, `InferenceRunner` **every epoch**. What if I'd like to log (tensorboard) training-summaries more often whereas log validation-summaries every epoch?
 
For example, as I remember, `tf.slim` manages  `log_steps`, `steps_per_epoch` (may not be exact names). In Tensorpack, I am currently just setting small number for `steps_per_epoch` (will act like `log_steps`), and instead, setting large number for `max_epoch` for more frequent tensorboarding, and using `PeriodicTrigger` for validation-related-summaries. 

(Then, actually, Tensorpack's `steps_per_epoch` looks more like `log_steps` of `tf.slim`. I don't have any complaints about this, but just make sure if my understanding is correct.)

2. Is there a way to print training loss with some interval? Let's say my `steps_per_epch=100` and I intend it to update tensorboard every ~1hr. Meanwhile a progess bar is going (e.g., `52%|######3   
 |XX/YY[40:12<38:31, 1.34it/s]`, is there a way to visualize train loss more frequently?

For example,  as in KERAS 
````
Epoch 4/70 
800/800 [=====  ] - 401s 501ms/step -loss: 1.3346 -xentropy: 1.3346**
````
The `loss` and `xentropy` parts keep updating without newline. Was there a such callback (i.e, in-the-middle-of-epoch printing) in tensorpack? Sometimes, within ~1hr-tensorbaord-logging-period, I want to feel if a loss looks decreasing instead of having no idea.

Thanks for the awesome library and hope this project goes long.

**Comment by ywpkwon** [comment]
Thanks.  I tried to add `ProgressBar`
````
return AutoResumeTrainConfig(
        dataflow=dataset_train,
        callbacks=[
            ModelSaver(keep_checkpoint_every_n_hours=5),
            GPUUtilizationTracker(),
            EstimatedTimeLeft(),
            ProgressBar(names=['cost']),
        ],
        model=Model(num_classes=NUM_CLASSES),
        steps_per_epoch=steps_per_epoch,
        max_epoch=500,
    )
```

Now, it shows two progress bars, maybe one is the default one.
````
 88%|##############################################################6           |88/100[00:32<00:04, 2.44it/s], cost=0.156
 88%|####################################################################         |88/100[00:32<00:04, 2.44it/s]
````
How can I make the default one to print out `cost`?

**Comment by ppwwyyxx** [comment]
It's already in the document of TrainConfig.
